src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -85,7 +86,8 @@ def __init__(self, jobID=None, jobReport=None):
         self.log.showHeaders(True)
 
         # self.root is the path the Wrapper is running at
-        self.root = os.getcwd()
+        self.root = Path.cwd()
+        self.jobIDPath = self.root
```

Comment:
It's probably less error-prone if it is simply 

```suggestion
        self.jobIDPath = None
```

Reply:
Why do you think it would be less error-prone?

I did not apply this comment, but I added a comment to explain why I am setting `self.jobIDPath` as `self.root` by default.

Basically, in the context of resources with no external connectivity, the `PJA` first initializes a `JobWrapper` with a `jobID`, so the pre-processing phase creates a directory `self.root/self.jobID` in the vobox where inputs of the job will be located. 

The content of this directory is then sent to a remote computing resource. From there, a new `JobWrapper` is initialized to process the executable this time. If I would specify the `jobID` in this `JobWrapper`, inputs would not be found because they would be included in `self.root` of the worker node. Therefore, I don't provide any `jobID` when the `JobWrapper` is initialized (because it is anyway not needed to process the executable).

Here is a schema to highlight the issue if `self.jobID` is provided to `JobWrapper` in the remote computing worker node with no external connectivity: 
![pjaDirectories](https://github.com/user-attachments/assets/6983cd7f-4104-48e0-814f-d00748954b5d)